src/components/common/ColonyHome/ColonyHomeInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import ColonyTitle from './ColonyTitle';
+import ColonyNavigation from './ColonyNavigation';
+
+import styles from './ColonyHomeLayout.css';
+
+const displayName = 'common.ColonyHome.ColonyHomeInfo';
+
+interface Props {
+  isMobile: boolean;
+  showNavigation: boolean;
+}
+
+const ColonyHomeInfo = ({ isMobile, showNavigation }: Props) => (
+  <aside className={styles.leftAside}>
+    <ColonyTitle />
+    {!isMobile && showNavigation && <ColonyNavigation />}
```

Comment:
@willm30 after updating `ColonyHomeLayout` to use `ColonyHomeInfo` in the non-mobile case, isMobile still makes sense here. I removed showNavigation because it was true in all cases.